content/building-blocks/automate-and-execute-your-work/reproducible-work/google_cloud_docker.md

```diff
@@ -0,0 +1,155 @@
+---
+title: "Import and Run a python environment on Google cloud with Docker" 

```

Comment:
I feel like we should make 2 separate tutorials: one for Windows (and say it is explicitly for Windows) and one for the cloud (and by extension for Unix systems). Were you thinking of adding Google cloud to this same page @DiSanchz ? I think it'll be too long as a tutorial then. Will be great to hear from @shrabasteebanerjee on this issue too.

Reply:
My intention for this week was indeed to make the modifications derived from the feedback and then add the google cloud segment on top.  In any case I also had this impression that its extension might be above average (once the google cloud section is added) when compared with other building blocks already available in TSH.